<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminDashboard</class>
 <widget class="QMainWindow" name="AdminDashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1184</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-20</y>
      <width>200</width>
      <height>641</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(250, 250, 250);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QPushButton" name="button1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: none;
    background-color: rgb(250, 250, 250);   /* default */
    color: black;
    padding: 8px 16px;
    border-radius: 20px;
}

/* Hover khi chưa checked */
QPushButton:hover:!checked {
    background-color: rgb(235, 235, 235);   /* hover nhẹ */
}

/* Khi nhấn */
QPushButton:pressed {
    background-color: rgb(210, 210, 210);
}

/* Khi checked (đang chọn) */
QPushButton:checked {
    background-color: rgb(200, 200, 200);
}

/* Hover khi đang checked (clicked) */
QPushButton:checked:hover {
    background-color: rgb(190, 190, 190);  /* đậm hơn chút */
}
</string>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>180</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: none;
    background-color: rgb(250, 250, 250);   /* default */
    color: black;
    padding: 8px 16px;
    border-radius: 20px;
}

/* Hover khi chưa checked */
QPushButton:hover:!checked {
    background-color: rgb(235, 235, 235);   /* hover nhẹ */
}

/* Khi nhấn */
QPushButton:pressed {
    background-color: rgb(210, 210, 210);
}

/* Khi checked (đang chọn) */
QPushButton:checked {
    background-color: rgb(200, 200, 200);
}

/* Hover khi đang checked (clicked) */
QPushButton:checked:hover {
    background-color: rgb(190, 190, 190);  /* đậm hơn chút */
}
</string>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>310</y>
       <width>200</width>
       <height>50</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
    border: none;
    background-color: rgb(250, 250, 250);   /* default */
    color: black;
    padding: 8px 16px;
    border-radius: 20px;
}

/* Hover khi chưa checked */
QPushButton:hover:!checked {
    background-color: rgb(235, 235, 235);   /* hover nhẹ */
}

/* Khi nhấn */
QPushButton:pressed {
    background-color: rgb(210, 210, 210);
}

/* Khi checked (đang chọn) */
QPushButton:checked {
    background-color: rgb(200, 200, 200);
}

/* Hover khi đang checked (clicked) */
QPushButton:checked:hover {
    background-color: rgb(190, 190, 190);  /* đậm hơn chút */
}
</string>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>-10</y>
      <width>1001</width>
      <height>641</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
